core/scss/settings/settings.scss

```diff
@@ -4926,6 +4926,11 @@ $daypicker-day-width: 22px !default;
 $datepicker-large-width: $datepicker-width + 25px !default;
 $daypicker-large-day-width: 30px;
 
+$drawer-border-color: $md-gray-10;
+$drawer-background-color: $white;
+$drawer-border-radius: 2px;
+$drawer-zindex: 1000;
```

Comment:
This needs to fit in our z-index variables - but need to find out from design if it's intended to be above Topbar but below popover? 
```
$zindex-sidebar:              1000;
$zindex-top-bar:              1005;
$zindex-close:                1015;
$zindex-popover:              1020;
$zindex-tooltip:              1030;
$zindex-top-bar-fixed:        1030;
$zindex-modal-background:     1040;
$zindex-modal:                1050;
```

Reply:
yea .. there was no guidance on this .. meant to ask I would think you would want a tooltip or popover to work inside and a modal to go over ..

Reply:
@pauljeter  what should the zindex be for drawer?

Reply:
I think that it would be the same as the sidebar, since that is the component it is replacing.